### PR TITLE
Wip new

### PR DESCRIPTION
**Pull request by joshuabamboo**
@AnnJohn @jmburges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,119 @@
...
+####Connect to DB
+This is how we would connect to a database:
+
+```ruby
+ActiveRecord::Base.establish_connection(
+  :adapter => "sqlite3",
+  :database => "db/students.sqlite"
+)
+```
+####Create a table
```
Comment (jmburges) [inline README.md:34]: @joshuabamboo can you look into doing Active Record without Migrations? Pretty much can we just create the database ourselves? If not then we need to explain how to run these migrations without Rack and such

**Comment by jmburges** [comment]
back at you. If we can create a table using standard SQL I'd rather do that. They don't know how to run migrations yet

**Comment by joshuabamboo** [comment]
@jmburges should be migration free. :shipit: ?

**Comment by jmburges** [comment]
:shipit:

**Pull request merged**